### PR TITLE
LPS-96370 and LPS-96371

### PR DESCRIPTION
**Pull request by matethurzo**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@matethurzo I sent it back to @nhpatt at https://github.com/nhpatt/liferay-portal/pull/139

The change to Response doesn't make sense to me. We used to have delete return a Response, but then we did a mass change to remove it. I dont' remember why - but @nhpatt should have an idea for it.

@nhpatt can you go through what @matethurzo is trying to do and remove the Response return since we had removed it from all of our other apis?

**Comment by nhpatt** [comment]
Yes, we've been talking to see his needs. 

We removed the Response because we didn't support 204 return for void responses (we do now) and because Response is hard to parse with a client.
